addons/main/REVIVE/fn_clientInit.sqf

```diff
@@ -54,9 +54,16 @@ DFUNC(SelfCare) =
     // Var setzen
     player setVariable ["OPT_isUnconscious", 0, true];
     player setVariable ["OPT_isStabilized", 0, true];
+    player setVariable ["OPT_isOnMap", time, true];
```

Comment:
Bei einem Variablennamen mit "is" am Anfang würde man eigentlich einen boolschen Wert vermuten.